### PR TITLE
feat(3PT): Added support for GoogleRemarketing 

### PR DESCRIPTION
**Pull request by ashishpuliyel**
Google Remarketing tags as a thirdparty tag

## What does this change?
Adds support for google remarking tags and types as required to support them

## Why?
Part of the ticket [commjs] Get familiar with codebase

### CODE REVIEW

File: src/third-party-tags/remarketing.spec.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { remarketing } from './remarketing';
+
+describe('remarketing', () => {
+	it('should have expected properties', () => {
+		expect(remarketing({ shouldRun: true })).toMatchObject({
```
Comment (mxdvl) [inline src/third-party-tags/remarketing.spec.ts:5]: Better to use `toMatchObject` than `strictlyEqual`

**Review by mxdvl — APPROVED** [review]

**Comment by mxdvl** [comment]
Part of the discussion in the original PR: #39

**Comment by sndrs** [comment]
just realised - don't forget to add the export to the index file https://github.com/guardian/commercial-core/blob/main/src/index.ts

**Pull request merged**